src/classes/CRLP_RollupProcessingOptions.cls

```diff
@@ -38,13 +38,26 @@
 public class CRLP_RollupProcessingOptions {
 
     /**
-     * @description The processing mode for the calling batch operation - normal or LDV mode. In LDV mode,
+     * @description Defines the type of rollups being executed by the specific batch job. This is used with the
+     * RollupFilterSetting enum to select the correct Rollup__mdt records to use for the entire rollup operation.
+     */
+    public Enum RollupType {
+        AccountHardCredit,
+        ContactHardCredit,
+        AccountSoftCredit,
```

Comment:
I still think we should name this something that reflects that these are actually contact soft credits that we're aggregating. Maybe AccountContactSoftCredit?

Reply:
Good idea